### PR TITLE
Problem: tx sending concurrency coupled with num_accounts

### PR DESCRIPTION
**Pull request by yihuang**
Solution:
- have a separate worker pool to send transactions

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

## Summary by CodeRabbit

- **New Features**
	- Introduced multithreading for enhanced transaction sending capabilities.
	- Added a new function to manage concurrent transaction processing.
	- Improved error handling and retry mechanisms during transaction sending.

- **Improvements**
	- Enhanced modularity by decoupling transaction creation from sending logic.
	- Simplified the transaction sending process for better scalability.

### CODE REVIEW

File: testground/benchmark/benchmark/sendtx.py [hunk 1]
```diff
@@ -31,7 +34,32 @@ def fund_test_accounts(w3, from_account, num_accounts) -> [Account]:
...
+    q = queue.Queue()
+
+    def worker():
+        w3 = web3.Web3(web3.providers.HTTPProvider("http://localhost:8545"))
+        while True:
+            raw = q.get()
+            if raw is None:
+                break
+            try:
+                w3.eth.send_raw_transaction(raw)
```
Comment (mmsqe) [inline testground/benchmark/benchmark/sendtx.py:50]:
```suggestion
                w3.eth.send_raw_transaction(raw.rawTransaction)
```

File: testground/benchmark/benchmark/sendtx.py [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
 
 import web3
 from eth_account import Account
 
-from .utils import export_eth_account, send_transaction, send_transactions
+from .utils import export_eth_account, send_transactions, sign_transaction
 
 TEST_AMOUNT = 1000000000000000000
 GAS_PRICE = 1000000000
+TX_SENDING_WORKERS = 1000
```
Comment (mmsqe) [inline testground/benchmark/benchmark/sendtx.py:12]: not sure if need make it configurable based on machine

Comment (yihuang) [reply testground/benchmark/benchmark/sendtx.py:12]: let's figure out where the bottleneck is first.

Comment (mmsqe) [reply testground/benchmark/benchmark/sendtx.py:12]: seems it brings an extra queue running tasks from the original ThreadPoolExecutor, which may cause overhead

**Comment by yihuang** [comment]
fixed by https://github.com/crypto-org-chain/cronos/pull/1587

**Pull request closed without merging**